spec/v1/CSN-Interop-Effective.schema.yaml

```
@@ -1312,9 +1312,12 @@ definitions:
     title: Value Object
```

title looks wrong, should be Enum Value
```suggestion
    title: Enum Value
```

---

Not sure if we reuse this. If it is reused, this is indeed an object stating a potential value. 
But if we only use it for enums, I would also opt for renaming it and creating a separate interface in case we need something else

---

to be consistent with all other `definition names` and their defined `title`s, we should probably change it

---

Ok, so we'll change it to `Enum Value` or `Enum Value Object`, which will result in a TS type interface name change (as they are based on title (curiously) instead of the key name